src/components/HistoryTable.vue

```diff
@@ -57,7 +57,13 @@
                         </q-td> -->
         <q-td key="token" :props="props">
           <div @click="copyText(props.row.token)">
-            {{ shortenString(props.row.token, 10, 40) }}
+            {{
+              props.row.token.slice(0, 8) +
+              "..." +
+              props.row.token.slice(100, 110) +
```

Comment:
Can we take a dynamic range here? (for example somewhere in the middle + 10?)

Reply:
idea implemented in #55 from origin/